docs/send-data/opentelemetry-collector/remote-management/st-with-secrets.md

```diff
@@ -0,0 +1,96 @@
+---
+id: st-with-secrets
+title: Setting Environment variable with secret values for Source Template
+sidebar_label: Setting env variables
+description: Steps for setting environment variable with secret value which can be used by source template at runtime in a remotely managed opentelemetry collector.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <meta name="robots" content="noindex" />
```

Comment:
This blocks the doc from being searchable on our site and search engines - just wanted to double-check this is intentional.

Reply:
No changes needed then. It'll be omitted from sidebars, won't come in our site search or search engines, but if you share the link, it'll be accessible to anyone.